src/core/column/column.text.js

```diff
@@ -1,7 +1,20 @@
 import ColumnView from 'core/view/view.column';
+import Column from './column';
+import TemplatePath from 'core/template/template.path';
+
+TemplatePath.register('text-cell', (template, column) => {
```

Comment:
Sometimes it'll be transformed to decorator =) Unfortunately babel6 doesn't support it out of the box.